integration/spark2/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -41,18 +41,11 @@ class QueryTest extends PlanTest {
 
 
   val rootPath = new File(this.getClass.getResource("/").getPath + "../../../..").getCanonicalPath
-  val storeLocation = s"$rootPath/examples/spark2/target/store"
-  val warehouse = s"$rootPath/examples/spark2/target/warehouse"
-  val metastoredb = s"$rootPath/examples/spark2/target/metastore_db"
+  val storeLocation = s"$rootPath/integration/spark2/target/store"
+  val warehouse = s"$rootPath/integration/spark2/target/warehouse"
+  val metastoredb = s"$rootPath/integration/spark2/target/metastore_db"
 
   val spark = {
-    // clean data folder
-    if (true) {
```

Comment:
It is not required now as drop table support is added in carbon spark 2 integration so drop table command can delete the store as well.